### ISSUE TITLE
Ability to set RemoteNickFormat for Matterbridge

### ISSUE THREAD

**Issue by markuman** [nextcloud/spreed#4279]
**Is your feature request related to a problem? Please describe.**

No

**Describe the solution you'd like**

When using this for single user setup, it may annoy others on the target chat system when they see always

```
[nctalk] <markuman> my message
[nctalk] <markuman> bla blabla
```

So it would be nice to disable/overwrite it.

**Describe alternatives you've considered**

Dirty patch it like this

```
sed -i 's/\[{PROTOCOL}\] <{NICK}> //g' lib/MatterbridgeManager.php
```

**Additional context**

![Screenshot_2020-10-04 Glowing Bear #ansible-aws Ansible + AWS https github com ansible community wiki aws](https://user-images.githubusercontent.com/3920157/95015680-872cad80-064e-11eb-9e40-50c4412f0c1f.png)

**Comment by nickvergessen** [comment]
Well you can also adjust your matterbridge config file in tmp

**Comment by markuman** [comment]
@nickvergessen  
`/tmp` is on most systems not persistent I think.  
And my setup is using docker. 
So I think I can automate it for my needs. But it would be nice to overwrite this settings/variables.

**Comment by nickvergessen** [comment]
Just ment its the better hack then adjusting the Talk source code

**Comment by markuman** [comment]
@nickvergessen 

> Just ment its the better hack then adjusting the Talk source code

True. But every click in the settings seems to overwrite the .toml file.  
I need to mount it as `:ro` in the container or do `chmod 444` on that file.

**Comment by markuman** [comment]
Uh, I figured out that the config is not persistent at all.

After restart   

```
cat /tmp/bridge-8j5ditpi.log 
time="2020-10-04T17:44:45Z" level=info msg="Running version 1.18.3 8b26e42a" prefix=main
time="2020-10-04T17:44:45Z" level=fatal msg="Failed to read configuration file: &os.PathError{Op:"open", Path:"/tmp/bridge-8j5ditpi.toml", Err:0x2}" prefix=config
```

But in the right sidebar, the information are still present. So I need to click "save" again and hit several times on "enabled" until it gets the status "running".

**Comment by leArthurDent** [comment]
I'd just like to say that I'd also like this feature very much. Sadly, the toml files are not persistent and I haven't found a way to tweak the matterbidge config, except for adjusting MatterbridgeManager.php.
